tests/testthat/test-style_sigfig.R

```diff
@@ -0,0 +1,18 @@
+context("test-style_sigfig")
+testthat::skip_on_cran()
+
+test_that("correct rounding near thresholds", {
+
+  expect_equal(
+    c(1.0001, 1, 0.9999, 0.90000) %>% style_sigfig(digits = 1),
+    c("1", "1", "1", "0.9")
+  )
+
+  expect_false(
+    style_sigfig(0.97, digits = 1) == "0.1"
```

Comment:
I am going to merge this now, but i think this line is meant to be 

`style_sigfig(0.97, digits = 1) == "1.0"`